app/containers/RoomTypeIcon/OmnichannelRoomIcon.tsx

```diff
@@ -49,7 +49,7 @@ export const OmnichannelRoomIcon = ({ size, style, sourceType, status }: IOmnich
 				<SvgUri
 					height={size}
 					width={size}
-					color={STATUS_COLORS[status || 'offline']}
+					color={STATUS_COLORS[status || 'offline'] ?? STATUS_COLORS.offline}
```

Comment:
Is it really necessary to do this?

Reply:
I did it for the stories, but I think I can remove it.